packages/containers/TeamList/TeamList.js

```diff
@@ -79,7 +81,7 @@ class TeamListContainer extends Component {
 
   handleClickEditTeam = teamId => event => {
     event.preventDefault();
-    event.stopPropagation();
+    //event.stopPropagation();
```

Comment:
Is this the useless code? Let's delete it.

Reply:
Forgot to uncomment after debug